packages/monorepo-builder/packages/testing/src/ComposerJson/ComposerJsonSymlinker.php

```diff
@@ -55,6 +55,8 @@ public function decoratePackageComposerJsonWithPackageSymlinks(
                 'options' => [
                     'symlink' => false,
                 ],
+                // since composer 2.0 - see https://getcomposer.org/doc/articles/repository-priorities.md#default-behavior
+                'canonical' => false,
```

Comment:
@samsonasik In the end it took more refactoring to find the bug.
This was the line, that was one of important fixes with new Composer 2.

This behavior changed: https://getcomposer.org/doc/articles/repository-priorities.md#default-behavior

Reply:
@TomasVotruba thank you